versioned_docs/version-0.x.0/kubeslice-cli/installing-kubeslice-cli.mdx

```diff
@@ -0,0 +1,50 @@
+# Installing the kubeslice-cli Tool
+
+This topic describes the steps to install the `kubeslice-cli` tool on different operating systems.
+
+## Installing kubeslice-cli on Windows
+
+To install kubeslice-cli on Windows:
+
+1. Download the latest Windows version from the [Releases](https://github.com/kubeslice/kubeslice-cli/releases) page.
+   After downloading, you can rename the executable file to **kubeslice-cli**.
```

Comment:
After downloading the latest version, you can ...

Reply:
I have mentioned "Download the latest windows version", saying it in the second sentence is redundant?

Reply:
After downloading sounds incomplete - what to download

Can you write - 

After downloading the file, you can rename it to ...